package.json

```diff
@@ -1,11 +1,13 @@
 {
-  "name": "express-graphql",
+  "name": "@chance-get-yours/express-graphql",
   "version": "0.9.0",
   "description": "Production ready GraphQL HTTP middleware.",
   "license": "MIT",
-  "private": true,
-  "main": "index.js",
-  "types": "index.d.ts",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
```

Comment:
Their publish strategy was garbage. Modified here and the build script so we can just `npm publish` in the root and it works.